Plugins/MeshFeatureTracking/src/MeshFeatureTrackingComponent.cpp

```diff
@@ -585,29 +585,30 @@ Scalar MeshFeatureTrackingComponent::getFeatureScale() const {
     auto ro = getRoMgr()->getRenderObject( m_pickedRoIdx );
     if ( m_pickedMesh->getRenderMode() == MeshRenderMode::RM_POINTS )
     { return ro->getAabb().sizes().norm() / 500; }
+    const auto& T = ro->getTransform();
     const auto& v = m_pickedMesh->getTriangleMesh().vertices();
     switch ( m_data.m_mode )
     {
     case PickingMode::VERTEX:
     {
         // return 1 fourth of the edge length of the first edge we can find with the vertex
-        const Ra::Core::Vector3& v0 = v[m_data.m_data[0]];
-        const Ra::Core::Vector3& v1 = v[m_data.m_data[1]];
+        const Ra::Core::Vector3& v0 = T * v[m_data.m_data[0]];
+        const Ra::Core::Vector3& v1 = T * v[m_data.m_data[1]];
         return ( v1 - v0 ).norm() / 4_ra;
```

Comment:
In this line and the following, you create temporary objects that you don't need. Your code could be rewritten as:
```c++
return ( T * v[m_data.m_data[1]] - T * v[m_data.m_data[0]] ).norm() / 4_ra;
```
Here you can notice that we compute the distance between two points transformed by a `Transform`, which is defined as
```c++
using Transform  = Eigen::Transform<Scalar, 3, Eigen::Affine>;
```
`Eigen::Affine` stores translation, rotation and scale. As applying a rotation or a translation to an object does not change the distances between its points, you don't have to multiply with the full matrix. 
According to [Eigen documentation](https://eigen.tuxfamily.org/dox/classEigen_1_1Transform.html#a257ab8b9141982bd6f56771ba822526f), you can use directly `T.linear()` directly. Thus your code becomes:
```c++
return ( T.linear() * v[m_data.m_data[1]] - T.linear() * v[m_data.m_data[0]] ).norm() / 4_ra;
```
Note that `linear()` build a reference expression and thus has no cost (https://eigen.tuxfamily.org/dox/classEigen_1_1Transform.html#title39)